docker/config.js

```diff
@@ -48,7 +52,7 @@ module.exports = {
 		"./plugins/survey/plugin.js",
 		"./plugins/graph/plugin.js",
 		"./plugins/hashcolor/plugin.js",
-		"./plugins/shield/plugin.js",
+		"./plugins/shield/plugin.js"
```

Comment:
Leave this goddamn trailing comma! This ain't JSON.

Reply:
Ok, I'll fix that